Week4/Lecture8/Assignment2/lecture8_assignment2/src/main/java/com/example/lecture8_assignment2/service/EmployeeService.java

```diff
@@ -0,0 +1,35 @@
+package com.example.lecture8_assignment2.service;
+
+import com.example.lecture8_assignment2.model.Employee;
+import com.example.lecture8_assignment2.repository.EmployeeRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class EmployeeService {
+
+    @Autowired
+    private EmployeeRepository employeeRepository;
```

Comment:
For best practice, we should use contructor dependency, pls refer leture 7/ assignment 2

Reply:
I have implemented it, please kindly check, thanks for the feedback!